pyccel/ast/variable.py

```diff
@@ -119,7 +119,7 @@ def __init__(
         is_kwonly=False,
         allows_negative_indexes=False
         ):
-        super().__init__()
+        super().__init__(name)
```

Comment:
Presumably you need to remove the `_name` attribute defined here?

Reply:
yes if this is finished by keeping symbol inheritance in Variable we can remove it